lib/src/components/supa_socials_auth.dart

```diff
@@ -346,6 +295,33 @@ class _SupaSocialsAuthState extends State<SupaSocialsAuth> {
 
         onAuthButtonPressed() async {
           try {
+            // Check if native Google login should be performed
```

Comment:
I think you can use sign in with google on android, iOS and web?

Reply:
Yeah, web sign in works either way, so I think it's fine, but feel free to fix the logic if you want to include the web to use the google_sign_in package as well!